environment.h

```diff
@@ -6,11 +6,11 @@
 #include "json.hpp"
 #include "utilities.h"
 
-static double env_func_1(std::vector<double> input){
+static double env_func_A(std::vector<double> input){
   return input[0];
 }
 
-static double env_func_2(std::vector<double> input){
+static double env_func_B(std::vector<double> input){
```

Comment:
Ahhh you switched back already! this is gonna cause merge conflicts with theother pull request! not a big deal, but make sure you are consistent across both.